docs-kits/kits/Eco_Pass_KIT/page-software-operation-view.md

```diff
@@ -0,0 +1,455 @@
+| AccessPolicyId | Policy that allows/restricts/enforces asset access constrains | ad8d2c57-cf32-409c-96a8-be59675b6ae5                                                                                                          |
+| ContractPolicyId | Policy that allows/restricts/enforces contract constrains | ad8d2c57-cf32-409c-96a8-be59675b6ae5                                                                                                          |
+
+#### Format and Fields:
+
+> **_INFO:_** *For testing purposes and in order to ease the access to your assets we are going to define the **same policy as accessPolicy and as contractPolicy**. However, you are recommended to configure two separated policies and specify them adapting each one of them to your specific needs.*
+
+```json
+{
+    "@context": {},
+    "@id": "{{ContractDefinitionId}}",
+    "@type": "ContractDefinition",
+    "accessPolicyId": "{{AccessPolicyId}}",
+    "contractPolicyId": "{{ContractPolicyId}}",
+    "assetsSelector" : {
+        "@type" : "CriterionDto",
+        "operandLeft": "https://w3id.org/edc/v0.0.1/ns/id",
+        "operator": "=",
+        "operandRight": "{{AssetId}}"
+    }
+}
+```
+
+### Digital Twin Registration
+
+Once you finish the configuration, to make the endpoint public configure your digital twin in the following way:
+
+ > **_INFO:_** *You need to be able to request tokens for the **Catena-X Central IAM** in order to **configure Digital Twins** in the Registry.*
+
+
+
+#### Variables:
+
+| Name | Description | Example Value |
+| ---- | -------- | ---- |
+| DigitalTwinId | Manually generated DID that contains a UUID | 32aa72de-297a-4405-9148-13e12744028a |
+| DigitalTwinSubmodelId | Sub Model Id registered in the Digital Twin Registry | 699f1245-f57e-4d6b-acdb-ab763665554a |
+| PartInstanceId | Battery passport DMC code or the part instance Id | X123456789012X12345678901234566 |
+| manufacturerPartId | The Part Id given by the manufacturer, it idenfies the type of the product | XYZ78901 |
+| EDCProviderUrl | URL to the endpoint which contains the EDC Provider | [https://edc.control.plane/](https://edc.control.plane/) |
+| BPN | OPTIONAL: The endpoint address can include a BPN number, which shall lead to the EDC Provider, and return the contracts when called from an EDC Consumer | BPNL000000000000 |
+| SubmodelIdShort | EXACT STRING REQUIRED: The submodel id of the battery passports needs to be exactly the string: "batteryPass" | **batteryPass** |
+| BammModelVersionId | The semantic version of the asset passport model, currently the  version v3.0.1 is used | urn:bamm:io.catenax.battery.battery_pass:3.0.1#BatteryPass |
+
+> **_INFO:_** *It is important that the "SubmodelIdShort" is set in the correct format and that the EDCProviderUrl points to an valid EDC Provider, that providers valid contracts configured in the structure defined here.*
```

Comment:
the last "providers", shall it mean "provides"?

Reply:
done